src/components/meta-data/index.test.jsx

```diff
@@ -0,0 +1,2019 @@
+/**
+ * this is for mocking node env
+ * will not have window attribute, testing ssr
+ * https://jestjs.io/docs/en/configuration.html#testenvironment-string
+ * @jest-environment node
+ */
+// import React from "react";
+import Enzyme, { shallow } from "enzyme";
```

Comment:
I went with Enzyme so that many of the tests could be reused from the Engine-theme-sdk repo, because I didn't want this ticket to block beta for any longer. But, I agree that would be a worthwhile improvement, so I went ahead and opened a new ticket to convert them from Enzyme to React testing library and to remove enzyme from the components repo: https://arcpublishing.atlassian.net/browse/THEMES-1231